cpp/environment/LocalBondProjection.cc

```diff
@@ -80,14 +80,8 @@ void LocalBondProjection::compute(box::Box& box,
     // Get the maximum total number of bonds in the neighbor list
     const size_t tot_num_neigh = nlist->getNumBonds();
```

Comment:
Currently `getNumBonds` returns an unsigned int, which would eliminate the need for the static_casting below.